Resources/Prototypes/_Mono/Entities/SpaceArtillery/Ammo/50cal.yml

```
@@ -0,0 +1,77 @@
+    sprite: _Mono/Objects/SpaceArtillery/50_armorpiercing_machinegun_casing.rsi
+    layers:
+    - state: base
+      map: ["enum.AmmoVisualLayers.Base"]
+
+# .50 Cal AP
+
+- type: entity
+  id: 50BulletArmorPiercing
+  name: .50 AP bullet
+  parent: 50BulletBase
+  categories: [ HideSpawnMenu ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)